packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -2689,6 +2693,9 @@ export class SnapController extends BaseController<
 
       this.#updateApproval(pendingApproval.id, {
```

Comment:
Please add a test or an assertion to an existing test testing that `initialConnections` is part of the `requestState`.

Reply:
Added.